sdk/playwrighttesting/Azure.ResourceManager.PlaywrightTesting/samples/Generated/Samples/Sample_PlaywrightTestingAccountCollection.cs

```diff
@@ -22,7 +22,7 @@ public partial class Sample_PlaywrightTestingAccountCollection
         public async Task CreateOrUpdate_AccountsCreateOrUpdate()
         {
             // Generated from example definition: 2024-12-01/Accounts_CreateOrUpdate.json
-            // this example is just showing the usage of "Account_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Accounts_createOrUpdate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `createOrUpdate`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/playwrighttesting/PlaywrightTesting.Management/main.tsp#L100